torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -619,6 +624,10 @@ class TORCH_API ProcessGroupNCCL : public ProcessGroup {
 
   // Counting for the sequential number of NCCL collective call.
   uint64_t seq_{0};
+
+  // ProcessGroupUCC shared library handle
+  std::shared_ptr<at::DynamicLibrary> ucc_lib;
```

Comment:
Does this need to be a member variable? Seems like it is only use locally in ProcessGroupNCCL's constructor?

Reply:
You are right. It is now only used in ProcessGroupNCCL's constructor. But I still prefer to leave it as a static member, because in the future, it might be used for more communications, such as:
```C++
ucc_lib_->sym("ucc_version");
ucc_lib_->sym("built_with_cuda");
// ... etc.
```